### ISSUE TITLE
Tray Applet Icon Size & Missing Issues

### ISSUE THREAD

**Issue by proximous** [linuxmint/cinnamon#9436]
```
 * Cinnamon version (cinnamon --version): 4.6.6
 * Distribution - Mint 20
 * Graphics hardware *and* driver used: NVidia (nvidia-driver-440)
 * 32 or 64 bit: 64
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 ```

**Issue**
On a fresh install of Linux Mint 20, tray applet icons appear correctly and are correctly sized after initial installation. Upon reboot, some icons never appear, or disappear, and some icons (Dropbox) are now small (not scaled for the HiDPI screen).

**Steps to reproduce**
Install apps like Dropbox, Teamviewer, Google Chrome

**Expected behaviour**
Icons present and sized correctly.

**Other information**

**Configuration info, if applicable**
I see lots of icon errors in the .xsession-errors log, but not sure how to interpret them.
[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/4886972/xsession-errors.txt)
![Screenshot from 2020-07-07 16-20-09](https://user-images.githubusercontent.com/8294191/86838078-c22e3d80-c06d-11ea-81ec-113bff0ebc3f.png)

**Comment by fermmm** [comment]
Same problem here, some icons do not appear but the application is running, so the icon should be there

**Comment by mtwebster** [comment]
Hi, if you wouldn't mind can you follow the directions in this comment:

https://github.com/linuxmint/xapps/issues/98#issuecomment-656700861

This should fix your issue.  Hopefully we'll get a new release out soon with this.

**Comment by fermmm** [comment]
> Hi, if you wouldn't mind can you follow the directions in this comment:
> 
> [linuxmint/xapps#98 (comment)](https://github.com/linuxmint/xapps/issues/98#issuecomment-656700861)
> 
> This should fix your issue. Hopefully we'll get a new release out soon with this.

That fixed the issue for me.

**Comment by fermmm** [comment]
> > Hi, if you wouldn't mind can you follow the directions in this comment:
> > [linuxmint/xapps#98 (comment)](https://github.com/linuxmint/xapps/issues/98#issuecomment-656700861)
> > This should fix your issue. Hopefully we'll get a new release out soon with this.
> 
> That fixed the issue for me.

I've tried in another computer with the same issue but in this case the problem is not fixed. The last version of Ferdi is not showing the tray icon: 
https://getferdi.com/download

**Comment by mtwebster** [comment]
I tried with the 5.5 .deb package from there, and I got an icon briefly before it disappeared the first time I ran it.  Every time since it's been fine:
![image](https://user-images.githubusercontent.com/262776/87236399-1e1efc00-c3b7-11ea-8773-0594b385d408.png)

The menu appears broken, however.  But it's broken in xfce also (which uses a different tray panel applet), so I think it's a problem with Ferdi.  Other electron apps work fine - slack for instance.

**Comment by fermmm** [comment]
> I tried with the 5.5 .deb package from there, and I got an icon briefly before it disappeared the first time I ran it. Every time since it's been fine:
> ![image](https://user-images.githubusercontent.com/262776/87236399-1e1efc00-c3b7-11ea-8773-0594b385d408.png)
> 
> The menu appears broken, however. But it's broken in xfce also (which uses a different tray panel applet), so I think it's a problem with Ferdi. Other electron apps work fine - slack for instance.

It works like you show in one of my computers, but in the other one it does not:

![2020-07-11_21-48](https://user-images.githubusercontent.com/3660373/87236454-524aea80-c3c0-11ea-9c59-6e0d8089c682.png)

**Comment by fermmm** [comment]
> I tried with the 5.5 .deb package from there, and I got an icon briefly before it disappeared the first time I ran it. Every time since it's been fine:
> ![image](https://user-images.githubusercontent.com/262776/87236399-1e1efc00-c3b7-11ea-8773-0594b385d408.png)
> 
> The menu appears broken, however. But it's broken in xfce also (which uses a different tray panel applet), so I think it's a problem with Ferdi. Other electron apps work fine - slack for instance.

Also when removing system tray applet and add it again it shows an exclamation mark:
![err](https://user-images.githubusercontent.com/3660373/87236496-d43b1380-c3c0-11ea-9fde-1140d9f1924e.png)

**Comment by mtwebster** [comment]
Is the XApp status applet running?  That's the one that should have this icon.

**Comment by fermmm** [comment]
> Is the XApp status applet running? That's the one that should have this icon.

No, it was not running, now it appears, I didn't know there was another applet to add, thanks!

**Comment by proximous** [comment]
> Hi, if you wouldn't mind can you follow the directions in this comment:
> 
> [linuxmint/xapps#98 (comment)](https://github.com/linuxmint/xapps/issues/98#issuecomment-656700861)
> 
> This should fix your issue. Hopefully we'll get a new release out soon with this.

Applying the DEBs in the link fixed things for me. Dropbox and other Icons are now the correct size, and the Chrome Icon is back.

When do you plan to make this fix official, so I don't have to worry about accidentally taking the update that will break it?

**Comment by proximous** [comment]
@mtwebster should I do something to prevent being offered the update to xapp that would make this issue reappear?
